cisco_aci/datadog_checks/cisco_aci/cisco.py

```diff
@@ -52,16 +52,32 @@ def check(self, instance):
             raise Exception("The Cisco ACI check requires at least one url")
 
         username = instance['username']
-        pwd = instance['pwd']
+        pwd = instance.get('pwd')
         instance_hash = hash_mutable(instance)
 
+        appcenter = _is_affirmative(instance.get('appcenter'))
+
+        cert_key = instance.get('cert_key')
+        if not cert_key and instance.get('cert_key_path'):
+            with open(instance.get('cert_key_path'), 'r') as f:
```

Comment:
The mode should be `rb` since https://cryptography.io/en/latest/hazmat/primitives/asymmetric/serialization/#cryptography.hazmat.primitives.serialization.load_pem_private_key expects `bytes`.

Reply:
it's weird that it works

Reply:
oh